src/components/AppNavigation/Settings.vue

```diff
@@ -256,8 +255,9 @@ export default {
 		},
 		defaultReminderOptions() {
 			const defaultAlarms = getDefaultAlarms().map(seconds => {
+				const label = seconds === 0 ? t('calendar', "at event's start") : t('calendar', "{name} before the event's starts", { name: moment.duration(Math.abs(seconds) * 1000).locale(this.locale).humanize() })
```

Comment:
Too much text for the width of the field unfortunately.

```suggestion
				const label = seconds === 0 ? t('calendar', "At event start") : moment.duration(Math.abs(seconds) * 1000).locale(this.locale).humanize()
```